src-user/Model/GroupManagerInterface.php

```diff
@@ -37,19 +37,15 @@ public function deleteGroup(GroupInterface $group);
 
     /**
      * Finds one group by the given criteria.
-     *
-     * @return GroupInterface
      */
-    public function findGroupBy(array $criteria);
+    public function findGroupBy(array $criteria): ?object;
 
     /**
      * Finds a group by name.
      *
      * @param string $name
-     *
-     * @return GroupInterface
      */
-    public function findGroupByName($name);
+    public function findGroupByName($name): ?object;
```

Comment:
Object?

Reply:
Should be GroupInterface, but for the /src-user folder I propose to not do any changes and just add it to the baseline?

Reply:
(this was already fixed)